app/scripts/api/config.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+module.exports = [
+    $stateProvider',
```

Comment:
This file is either not needed or breaks the app. You're missing proper quotes around `$stateProvider`